src/unix-manager.c

```diff
@@ -351,12 +351,7 @@ static int UnixCommandAccept(UnixCommand *this)
 
     /* read client version */
     buffer[sizeof(buffer)-1] = 0;
-    ret = recv(client, buffer, sizeof(buffer)-1, 0);
```

Comment:
why is this removed? Should be explained in the commit msg, and probably have its commit

Reply:
Should not be removed... thanks for catching